includes/wc-product-functions.php

```diff
@@ -64,6 +64,21 @@ function wc_get_product( $the_product = false, $deprecated = array() ) {
 	return WC()->product_factory->get_product( $the_product, $deprecated );
 }
 
+/**
+ * Get product class name.
```

Comment:
Should we document that an instance of `WC_Product_Simple` is returned if an invalid `$product_type` is passed? I'm not sure if that is the best behavior (and changing it is probably outside the scope of this PR) so I would suggest documenting. To me, it is not obvious that this is the behavior and it can cause some interesting bugs if, for example, there is a typo on the product type when calling this function.